client/app/shared/ss-card/ss-card.html

```diff
@@ -0,0 +1,9 @@
+<h3>{{ ::$ctrl.header }}</h3>
```

Comment:
We use vm everywhere, its predictable, understood.  Yes `$ctrl` should also be predictable, understood, but I got this funky feeling about mixing the use of  both, if we choose to go the `$ctrl` route, it should be one or the other in one huge swoop rather that bits and pieces (as this sends mixed messages as to the desired and accepted form of our code base; which brings up another thought entirely, updating the style guide to say as much)